tests/unit/parsers/test.manifestjson.js

```diff
@@ -5478,4 +5478,40 @@ describe('ManifestJSONParser', () => {
       ]);
     });
   });
+
+  describe('admin_install_only', () => {
+    it.each([
+      { applications: { gecko: { admin_install_only: true } } },
+      { applications: { gecko: { admin_install_only: false } } },
+      { browser_specific_settings: { gecko: { admin_install_only: true } } },
+      { browser_specific_settings: { gecko: { admin_install_only: false } } },
```

Comment:
Thought: how about including an additional one where we do have both gecko and gecko_android properties, strict_min_version/string_max_version properties in the gecko_android one and admin_install_only in the gecko property?

just to make sure that if we would be tweaking the implementation we couldn't mistakenly look for the admin_install_only property from gecko_android and instead we still hit the expected linting error.